doc/source/coregistration.md

```diff
@@ -579,17 +579,27 @@ methods, and create edge artefacts.
 :warning: This particular method relies on storing all data entirely on disk.
 ```
 
-Sometimes, we want to split a coregistration across different spatial block of an elevation dataset, running that
+Sometimes, we want to split a coregistration across different spatial blocks of an elevation dataset, running that
 method independently in each subset.
 It can also happen that memory requirements exceed the capacity of the computers.
 In such cases, it may be necessary to process the data in blocks and then aggregate the results afterward.
 
+> **Note:**
+> The `block_size_fit` parameter adjusts the size of the tiles over which the coregistration methods are computed.
+>
+> The `block_size_apply` parameter allows the DEM to be aligned in blocks to prevent memory leaks. In such case
+> reduce block_size_apply value.
```

Comment:
We should write "to reduce memory usage" instead of "to prevent memory leaks". 

Memory leaks is incorrect memory build-up in a program, e.g. when memory is allocated and not released later. Here it's all correctly allocated, we just decrease the peak memory usage by running smaller tiles :slightly_smiling_face: